src/controllers/playback/videoosd.js

```diff
@@ -1388,8 +1392,16 @@ define(["playbackManager", "dom", "inputManager", "datetime", "itemHelper", "med
 
                 case "mouse":
                     if (!e.button) {
-                        playbackManager.playPause(currentPlayer);
-                        showOsd();
+                        if (playerPauseClickTimeout) {
```

Comment:
Again here this check shouldn't be necessary.

Reply:
It would still need a way to check if the user has clicked though. Is there a better way?

Reply:
@thornbill

Reply:
`touch` has something similar
https://github.com/jellyfin/jellyfin-web/blob/637480230959217fade1c90542357bf0939f5b61/src/controllers/playback/videoosd.js#L1385-L1389

Reply:
Actually, I got almost the same
```js
case "mouse":
    if (!e.button) {
        if (now - lastPointerDown < 300) {
            clearTimeout(playerPauseClickTimeout);
        } else {
            playerPauseClickTimeout = setTimeout(function () {
                playbackManager.playPause(currentPlayer);
                showOsd();
            }, 300);
        }
        lastPointerDown = now;
    }

    break;
 ```